### ISSUE TITLE
[Question] `src` in attributes

### ISSUE THREAD

**Issue by LeGuit** [Netflix/fast_jsonapi#60]
Hey, 

Thanks for this gem !

I haven't see this yet in documentation but do you plan to add the `src` when you create an object e.g. :  
```
HTTP/1.1 201 Created
Location: http://example.com/photos/550e8400-e29b-41d4-a716-446655440000
Content-Type: application/vnd.api+json

{
  "data": {
    "type": "photos",
    "id": "550e8400-e29b-41d4-a716-446655440000",
    "attributes": {
      "title": "Ember Hamster",
      "src": "http://example.com/images/productivity.png"
    },
    "links": {
      "self": "http://example.com/photos/550e8400-e29b-41d4-a716-446655440000"
    }
  }
}
```

Same for links e.g.:
```
 "links": {
    "self": "http://example.com/articles/1"
}
```

**Comment by shishirmk** [comment]
We are still figuring out what our roadmap would be. We are overwhelmed by the response from the community. We are reviewing pull requests and figuring out what our 1.1.0 release would look like and when it would be out. 

If you could submit a pull request then we can certainly add it in sooner.

**Comment by shishirmk** [comment]
@LeGuit I cant find the `src` attribute being described in [jsonapi](http://jsonapi.org/format/#document-resource-identifier-objects) spec. It is being used an example but I am not sure if it is part of the spec.

The links feature has just landed in the `dev` branch let me know if that works for you.

**Comment by LeGuit** [comment]
Hey, 

You have `src` in Post request as in 
http://jsonapi.org/format/#crud-creating
Actually in the link you've send, it should be the presence of self. This way you can explore the API with those links.

**Comment by shishirmk** [comment]
@LeGuit Ability to add `self` and similar links per record has been released in version 1.2
